### PR TITLE
Join Button Fix

### PR DESCRIPTION
**Pull request by Geevies**
* Lobbysitter HUDs now properly update the Ready/Join button when the round starts.

### CODE REVIEW

**Comment by Geevies** [comment]
!bugfix

**Comment by Geevies** [comment]
!review

**Review by mikomyazaki — APPROVED** [review]

File: code/controllers/subsystems/ticker.dm [hunk 1]
```diff
@@ -486,6 +475,24 @@ var/datum/controller/subsystem/ticker/SSticker
...
+	for(var/obj/effect/landmark/start/S in landmarks_list)
+		//Deleting Startpoints but we need the ai point to AI-ize people later
+		if (S.name != "AI")
+			qdel(S)
+	// update join icon for lobbysitters
+	for(var/mob/abstract/new_player/NP in player_list)
+		if(!NP.client)
+			continue
+		var/obj/screen/new_player/selection/join_game/JG = locate() in NP.client.screen
+		JG.update_icon(NP)
```
Comment (Amunak) [inline code/controllers/subsystems/ticker.dm:490]: I wonder if it could happen that the button would not be found.

**Review by Amunak — APPROVED** [review]

File: code/controllers/subsystems/ticker.dm (hunk 1 above)
Comment (mikomyazaki) [reply code/controllers/subsystems/ticker.dm:490]: All `/mob/abstract/new_player`s are necessarily on the lobby screen, I think.

**Pull request merged**